make-sync-function

```diff
@@ -6,7 +6,7 @@ var path = require('path');
 var errorStatus = 1;
 var successStatus = 0;
 
-var workingDir = path.dirname(process.argv[1]);
+var scriptDir = path.dirname(process.argv[1]);
```

Comment:
To prevent further confusion over the purpose of this variable, I've renamed it to something more apt.